cleanlab/object_detection/rank.py

```diff
@@ -58,6 +62,11 @@
 def get_label_quality_scores(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
+    aggregation_weights: Optional[Dict[str, float]] = {
+        "overlooked": 0.6,
+        "swap": 0.2,
+        "badloc": 0.2,
+    },
```

Comment:
we said this should be None by default

Reply:
all of these 0.6, 0.2, 0.2 constants should also be imported from constants.py

Reply:
Fixed here: https://github.com/cleanlab/cleanlab/blob/b90724638d2eacdaacc3382c8bd2aa12643f47e9/cleanlab/object_detection/rank.py#L887